MyHeartCountsShared/MyHeartCountsShared.swift

```diff
@@ -0,0 +1,52 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import WatchConnectivity
+
+
+/// Key into a User Info dictionary transferred from the iPhone app to the Watch app.
+public struct InterDeviceUserInfoKey: RawRepresentable, Hashable, Sendable {
+    /// The key's raw value
+    public let rawValue: String
+    
+    /// Creates a new key
+    public init(rawValue: String) {
+        self.rawValue = rawValue
+    }
+}
+
+
+extension InterDeviceUserInfoKey {
+    /// `Bool` flag indicating that the watch should start a workout
+    public static let watchShouldEnableWorkout = Self(rawValue: "edu.stanford.MyHeartCounts.enableWatchWorkout")
+    /// `UInt8` value specifying which type of workout the watch should track.
+    ///
+    /// The value here corresponds to a `SpeziStudyDefinition/TimedWalkingTestConfiguration/Kind` raw value.
+    public static let watchWorkoutActivityKind = Self(rawValue: "edu.stanford.MyHeartCounts.watchWorkoutActivityKind")
+}
+
+
+extension WCSession {
+//    /// Sends a message and waits for a response.
+//    public func sendMessage(_ data: Data) async throws -> Data {
+//        try await withCheckedThrowingContinuation { continuation in
+//            self.sendMessageData(data) { response in
+//                continuation.resume(returning: response)
+//            } errorHandler: { error in
+//                continuation.resume(throwing: error)
+//            }
+//        }
+//    }
```

Comment:
Probably should be either removed or commented in; would be a good extension.

Reply:
removed for now but i kept the code; will become part of the Cross-Device-Communication module should we decide to implement that